core/src/main/scala/org/broadinstitute/dsde/rawls/util/AttributeSupport.scala

```diff
@@ -172,10 +171,13 @@ trait AttributeSupport {
    *
    * @param entity to update
    * @param operations sequence of operations
-   * @throws org.broadinstitute.dsde.rawls.workspace.AttributeNotFoundException when removing from a list attribute that does not exist
+   * @throws AttributeNotFoundException when removing from a list attribute that does not exist
    * @throws AttributeUpdateOperationException when adding or removing from an attribute that is not a list
    * @return the updated entity
    */
   def applyOperationsToEntity(entity: Entity, operations: Seq[AttributeUpdateOperation]): Entity =
     entity.copy(attributes = applyAttributeUpdateOperations(entity, operations))
 }
+
+class AttributeUpdateOperationException(message: String) extends RawlsException(message)
```

Comment:
This is the only place these are thrown, so it makes sense for them to be defined here.